src/bert_layers/configuration_bert.py

```diff
@@ -68,6 +67,8 @@ def __init__(
         rotary_emb_base: float = 10000.0,
         rotary_emb_scale_base=None,
         rotary_emb_interleaved: bool = False,
+        use_fa2: bool = True,
+        use_sdpa_attn_mask: bool = False,
```

Comment:
I think default should be True as False is not exact computation.

Reply:
I left the default as False since setting it to True means quadratic memory increase with sequence length on the backward pass.